### ISSUE TITLE
`api-evidence` POST user endpoint

### ISSUE THREAD

**Issue by apolkingg8** [SolomonDefi/solomon-monorepo#215]
The `POST /api/users` in document described as `User SignUp`, but in the `users.py` is `summary='Create User (Admin)'`.
It fails with 401 when I try it, so I guess there is some auth on this endpoint, but it should not have auth if it's an endpoint for signup. 

The tests around `/api/users` are blocked by this, and I can't find an example in python tests (looks like this part has no test). Could you take a look @solomondefi-dev ?

**Comment by solomondefi-dev** [comment]
Yes, the user create/edit/delete methods are restricted to Admin currently in the evidence API. The intent was to have a separate non-admin API, but it's not critical. Instead we can change the endpoint to be restricted to people who have made purchases via the dispute API.
